Mailer/app/api/v1/Handlers/Users/SubscribeHandler.php

```diff
@@ -63,21 +66,21 @@ public function handle($payload)
 
 
     /**
-     * Validate and load list from $params
+     * Validate and load list from $payload
      *
-     * @param $params
+     * @param $payload
      * @return ActiveRow $list - Returns mail list entity.
      * @throws InvalidApiInputParamException - Thrown if list_id or list_code are invalid (code 400) or if list is not found (code 404).
      */
-    protected function getList($params): ActiveRow
+    protected function getList($payload): ActiveRow
     {
-        if (isset($params['list_code'])) {
-            $list = $this->listsRepository->findByCode($params['list_code'])->fetch();
+        if (isset($payload['list_code'])) {
+            $list = $this->listsRepository->findByCode($payload['list_code'])->fetch();
         } else {
-            $listID = filter_var($params['list_id'], FILTER_VALIDATE_INT);
+            $listID = filter_var($payload['list_id'], FILTER_VALIDATE_INT);
```

Comment:
This kind of validation shouldn't be necessary anymore. Any validation should be performed by JSON schema.

Reply:
Yes, I removed it on other places, I must have missed this one.